Fabric.Identity.IntegrationTests/SuppressExecutionContextHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fabric.Identity.IntegrationTests
+{
+    public class SuppressExecutionContextHandler : DelegatingHandler
+    {
+        public SuppressExecutionContextHandler(HttpMessageHandler innerHandler) : base(innerHandler)
+        {
+        }
+
+        protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)

```

Comment:
What is this class doing?

Reply:
Without this class (and the use of it in the Integration Test Fixture), the integration tests all break in the middleware. The TestServer clears the HttpClient from the HttpContext, so when the middleware tries to access it, the server crashes with a Null Reference Exception. This is a known breaking change caused by .Net Core 2.2.